sdk/test-utils/perf/src/options.ts

```diff
@@ -167,6 +167,34 @@ export function parsePerfOption<TOptions>(
   return result as Required<PerfOptionDictionary<TOptions>>;
 }
 
+/**
+ * Validate that the provided command-line options are all recognized, throwing an error if an unrecognized
+ * option is provided.
+ *
+ * @param options A dictionary of options which should be passed.
+ */
+export function validateOptions<TOptions>(options: PerfOptionDictionary<TOptions>): void {
```

Comment:
Wondering if we should have unit tests for the perf framework with mocha just like every other package/lib in the repo to make sure there are no regressions in such methods.
Can you log an issue to track that work?

Reply:
You got it